src/app/search/basic-search/basic-search.component.ts

```diff
@@ -49,6 +51,9 @@ export class BasicSearchComponent extends Base implements OnInit {
    * @memberof BasicSearchComponent
    */
   openAdvancedSearch(): void {
-    this.searchService.setShowModal(true);
+    this.matDialog.open(AdvancedSearchComponent, {
+      width: '600px',
```

Comment:
Pixels

Reply:
Replaced the hardcoded width with the bootstrapModalSize.